tests/functional/contact.js

```diff
@@ -0,0 +1,133 @@
+        test.assertExists('.category-tabs > li[data-id="spaces"].current', 'Spaces tab is selected');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Contact, Communities elements: ' + communitiesUrl, 10, function suite(test) {
+
+    casper.start(communitiesUrl, function() {
+        test.assertHttpStatus(200);
+        test.assertExists('#map', 'Map exists');
+        test.assertElementCount('.category-tabs li a', 2, 'Tab navigation exists');
+        test.assertVisible('#map .legend', 'Communities legend is visible');
+        test.assertExists('#nav-spaces', 'Spaces navigation exists');
+        test.assertNotVisible('#nav-spaces', 'Spaces navigation is not visible');
+        test.assertExists('#nav-communities', 'Communities navigation exists');
+        test.assertVisible('#nav-communities', 'Communities navigation is visible');
+        test.assertNotVisible('#nav-spaces-select', 'Mobile navigation is not visible');
+        test.assertExists('.category-tabs > li[data-id="communities"].current', 'Communities tab is selected');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Contact, Spaces navigation click: ' + spacesUrl, 2, function suite(test) {
+
+    casper.options.waitTimeout = 10000;
+
+    casper.start(spacesUrl, function() {
+        this.click('#nav-spaces li[data-id="mountain-view"] > a');
+    });
+
+    casper.waitForUrl(/mountain-view/, function() {
+        test.assertUrlMatch(/mountain-view/, 'URL changed sucessfully');
+    });
+
+    casper.waitForSelector('#entry-container #mountain-view', function() {
+        test.assert(true, 'Page content updated successfully');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Contact, Tab navigation click: ' + spacesUrl, 3, function suite(test) {
+
+    casper.options.waitTimeout = 10000;
+
+    casper.start(spacesUrl, function() {
+        this.click('.category-tabs li[data-id="communities"] > a');
+    });
+
+    casper.waitForUrl(/communities/, function() {
+        test.assertUrlMatch(/communities/, 'URL changed sucessfully');
+        test.assertVisible('#map .legend', 'Communities legend is visible');
+    });
+
+    casper.waitForSelector('#entry-container #communities', function() {
+        test.assert(true, 'Page content updated successfully');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Contact, Mobile interaction: ' + spacesUrl, 4, function suite(test) {
+
+    casper.options.viewportSize = config.viewport.mobile;
+    casper.options.waitTimeout = 10000;
+
+    casper.start(url, function() {
+        test.assertNotVisible('#map', 'Map is not visible on mobile');
+        test.assertVisible('#nav-spaces-select', 'Mobile navigation is visible');
```

Comment:
Seeing that you check that the mobile navigation is not visible on desktop:
https://github.com/mozilla/bedrock/pull/3092/files#diff-291e556dd45b5f79745989905a959658R24

Perhaps we should here ensure that the desktop navigation is not visible on mobile, for both spaces and communities? Or perhaps just ensuring that it is not visible for the one, will be enough.